src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import './Image.scss';
+
+// TODO Add loading on image onLoad
+
+export const ImageComponent = memo(({ src, size, onClick }) => {
+
+    const imageOnClick = useCallback(() => onClick(), [onClick]);
```

Comment:
also you didn't even imported the memo and useCallback ;-;

Reply:
yep, my mistake. Yesterday I didn't think